package.json

```diff
@@ -487,6 +487,14 @@
 					"command": "cosmosDB.openInPortal",
 					"when": "view == cosmosDBExplorer && viewItem == cosmosDBGraphAccount"
 				},
+				{
+					"command": "cosmosDB.openInPortal",
```

Comment:
IMO its more confusing than valuable if the open in portal links to the Cosmos DB Account instead of directly to the stored procedure

Reply:
Whoops, shouldn't be in there.